doc/roadmap.md

```diff
@@ -1,3 +1,19 @@
+<!---
+Copyright 2018-2021 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
 ## Roadmap for Palisade
 
 #### Near term
```

Comment:
I think the "Long Term" roadmap might need a small tweak. Mentions the ability for the Palisade Service to update policies. Is this correct, would this be the service that did this?